pydoctor/model.py

```diff
@@ -387,7 +479,23 @@ def report(self, descr: str, section: str = 'parsing', lineno_offset: int = 0) -
             section,
             f'{self.description}:{linenumber}: {descr}',
             thresh=-1)
+    
+    @property
+    def aliases(self) -> List['Attribute']:
+        """
+        Return the known aliases of an object. 
 
+        @note: It seems that the list is not always complete, though.
+        """
+        aliases: List['Attribute'] = []
+        for alias in filter(lambda ob: ob.kind is DocumentableKind.ALIAS and isinstance(ob, Attribute), 
+                         self.system.allobjects.values()):
+            assert isinstance(alias, Attribute)
+            assert alias.parent is not None
+            if alias.parent._resolveDocumentable(alias) == self.fullName():
+                aliases.append(alias)
+        return aliases
```

Comment:
This is not the right implementation, the aliases should be computed at post-processing time.